includes/data-port/models/class-sensei-import-question-model.php

```diff
@@ -125,15 +125,16 @@ private function sync_meta() {
 			}
 
 			if ( is_wp_error( $new_value ) ) {
-				return new WP_Error(
-					'sensei_import_question_meta_field_invalid',
+				$this->add_line_warning(
 					sprintf(
 						// translators: First placeholder is name of field, second placeholder is error returned.
 						__( 'Meta field "%1$s" is invalid: %2$s', 'sensei-lms' ),
```

Comment:
We've talked about making this a more user-friendly error, but I thought it could be done in a different PR.